tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-test-ci-pipeline.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             --cwd /build/RelWithDebInfo \
     displayName: 'Run orttraining_ortmodule_tests.py'
     condition: succeededOrFailed()
-    timeoutInMinutes: 30
+    timeoutInMinutes: 60
```

Comment:
Out of curiosity, does the 60 minutes here include or exclude the build? (I'd be amazed that our tests could be taking > 30 minutes to run, and if so it might be worth creating some work items to look at reducing this.)

Reply:
include the build, the build itself is ~20 mins.

Reply:
One thing I noticed recently is that due to the shared allocator compilation is being run on each and every unit test. My observation was that this was causing significant increase in the test time. Each test on average is now taking 16-17 seconds. And that is why this increase in timeout is required.

Reply:
You can turn off the shared allocator for most tests and keep it on for some to test the code paths. There is a toggle switch in ORTModule.